docs/source/algorithms/MonitorEfficiencyCorUser-v1.rst

```diff
@@ -11,22 +11,20 @@ Description
 
 This algorithm normalises the neutron counts by monitor counts with an additional efficiency correction.
 
-To date this algorithm only supports the TOFTOF instrument.
-
 The monitor counts is the total count and it is stored in the SampleLogs of the input workspace.
 
-This count is corrected taking into account the monitor efficiency. The formula used for the correction is stored in the Parameters file and requires the incident energy (Ei), which is stored in the SampleLogs of the input workspace.
+This count is corrected taking into account the monitor efficiency. The formula used for the correction is stored in the Parameters file and requires the incident energy (:math:`E_i`), which is stored in the SampleLogs of the input workspace.
 
 The corrected value of the monitor counts is used to normalise the input workspace.
 
 
 Restrictions
 ###################################
 
-A formula named "formula\_eff" must be defined in the instrument
-parameters file. It is defined as "monitor counts * sqrt(Ei/25.3)"
+- A formula named "formula\_eff" must be defined in the instrument parameters file. For TOFTOF and DNS instruments it is defined as :math:`M*\sqrt{\frac{E_i}{25.3}}`. The incident energy :math:`E_i` and the monitor counts :math:`M` are read in the sample logs of the input workspace.
```

Comment:
May I suggest removing the '*' from the efficiency formula? Or try something like '\cdot' if you really want to show a multiplication operator there.